.github/workflows/test.yml

```diff
@@ -181,6 +181,7 @@ jobs:
   test_api:
```

Comment:
Shall we rename these actions to `test_e2e_api` and `test_unit_api` to make them more specific for the feature reader?

Reply:
I guess we can rename `test_e2e_api` but `test_unit` runs not only for API as it seems to me